### PR TITLE
[Payments] E2E tests - audit logs for provider created

### PR DESCRIPTION
**Pull request by peschina**
### Ticket:

- [21015](https://dev.azure.com/OGCIO-Digital-Services/Digital%20Services%20Programme/_workitems/edit/21015)

### Description

Added E2E test to make sure an audit log event is created when a new provider is created.
Changed metadata for provider creation to agreed structure. The same change will be implemented for other occurrences of audit log event creation.
Changed `manualBankTransfer` fixture to return name and id of the provider. The same logic will be applied to other resources created in fixtures if needed
## Type

- [ ] **Dependency upgrade**
- [ ] **Bug fix**
- [ ] **New feature**
- [x] **Dev change**
- [x] **Additional tests**
- [ ] **Documentation**
- [ ] **Other**

### Checklist:

- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] This change might impact the developer experience of others and should be communicated

### Screenshots:

N/A

### CODE REVIEW

File: apps/payments/e2e/utils/constants.ts [hunk 1]
```diff
@@ -91,3 +91,35 @@ export const paymentRequestValidationErrorTexts: Record<
 };
 
 export const SAFE_PID_FILENAME = "ngrok.pid";
+
+export enum AuditLogEventType {
```
Comment (nnorbert) [inline apps/payments/e2e/utils/constants.ts:95]: These were defined in the auditLogEvents file, which I can't see that it was deleted or not. Won't this code be duplicated?

Comment (peschina) [reply apps/payments/e2e/utils/constants.ts:95]: They are in `payments-api,` not `payments` :)

**Review by rgrnearform — APPROVED** [review]

File: apps/payments/e2e/utils/constants.ts (hunk 1 above)
Comment (nnorbert) [reply apps/payments/e2e/utils/constants.ts:96]: I see. Thank you!

**Review by nnorbert — APPROVED** [review]
LGTM! The events duplication can remain for now, but we should find a better solution to not copy all the existing events from payments-api into payments. But that can be in a separate PR.

**Pull request merged**